src/main/java/com/google/codeu/servlets/FormHandlerServlet.java

```diff
@@ -172,6 +172,11 @@ private void handleEatenFood(HttpServletRequest request, HttpServletResponse res
         if (blobKey == null) {
             out.println("<p>You ate " + meal.getAmount() + " of "  + meal.getFood() +
                     " on " + format.format(meal.getDate()) + ".</p>");
+            if(meal.getCO2()==0){
+              out.println("This Food is currently not in our Database. Feel free to add its CO2 information to our CO2 Data Form!");}
+            else{
+            out.println("<p> It produces" + meal.getCO2() + "CO2 in total.</p>");
```

Comment:
Here and below, do you need some spacing around the co2 number? This looks like it will say `It produces7CO2 in total`